src/components/Apps/ChatAi/components/ChitChat/ChatInput.vue

```diff
@@ -84,7 +106,7 @@ export default {
       this.inputValue = ''
     },
     onSelectChange(value) {
-      this.$emit('select-prompt', value)
+      this.$emit('select-model', value)
     }
   }
 }
```

Comment:
There aren't any major discrepancies identified in your provided template for Vue.js components. However, some adjustments might need to be made based on project specifications and specific needs.

For instance:

1. Ensure consistency with the existing documentation and standards of each framework/library you integrate.
2. Update component-level documentation, if necessary, reflecting new changes like the `isLoading` property's conditionals.
3. In case of external libraries, ensure compatibility issues have not been overlooked.
4. Maintain an updated version control history to prevent unexpected dependencies or conflicts.

If none of these details seem correct from this snippet alone, please send more detailed information about what needs checking specifically.